CONTRIBUTING.md

```diff
@@ -125,41 +148,62 @@ Please follow these steps to have your contribution considered by the maintainer
 2. Follow the [styleguides](#styleguides)
 3. After you submit your pull request, verify that all [status checks](https://help.github.com/articles/about-status-checks/) are passing <details><summary>What if the status checks are failing?</summary>If a status check is failing, and you believe that the failure is unrelated to your change, please leave a comment on the pull request explaining why you believe the failure is unrelated. A maintainer will re-run the status check for you. If we conclude that the failure was a false positive, then we will open an issue to track that problem with our status check suite.</details>
 
-While the prerequisites above must be satisfied prior to having your pull request reviewed, the reviewer(s) may ask you to complete additional design work, tests, or other changes before your pull request can be ultimately accepted.
+Before a PR can be merged it must pass all of the automated tests and also be reviewed by two maintainers. All of the above requirements must be met before your pull request will be reviewed. Please be aware that the the reviewers may ask you to complete additional design work, tests, or other changes before your pull request can be ultimately accepted.
+
+### Writing Documentation
+
+Great code is only one half of any successful project, and great documentation is just as important. The reality is that open source projects can stand or fall based on the quality of their documentation.
+
+The documentation for FOSSBilling is hosted here: [FOSSBilling Docs](https://docs.fossbilling.org/)
+
+Documentation is buit using [Docusauras](https://docusaurus.io/) from this [GitHub repository](https://github.com/FOSSBilling/docs). You can contribute directly to the repo on GitHub or using the *Edit this page* links on each page of the docs site.
+
+Please try to be thorough and clear when writing directions. Something might seem obvious to you, but do not assume that it is to everybody else. 
+
+### Translating FOSSBilling
+
+We would like FOSSBilling to be available to as many people in as many languages as possible. 
+
+The software is primarily written in English. If you are a native or fluent speaker of another language then we could really use your help with transaltionn. 
+
```

Comment:
Link to crowding / translate.fossbilling.com

Reply:
Also transaltionn should be transaltion

Reply:
> Also transaltionn should be transaltion

Fixed.